doc/source/changes/version_0_26.rst.inc

```diff
@@ -254,5 +254,20 @@ Fixes
 
 * fixed array creation with axis(es) given as string containing only one label (axis name and label were inversed).
 
+* fixed extracting LArray object from a CSV or Excel file when second axis is implicit (closes :issue:`372`).
+  For example, let say we want to read a CSV file 'pop.csv' with content ::
```

Comment:
s/let say/let us say/ or /let us suppose/ or /let's say/...
s/with content/with the following content (indented for clarity)/